### ISSUE TITLE
Add undo support for the queue

### ISSUE THREAD

**Issue by CreamyCookie** [quodlibet/quodlibet#1986]
Let's say you accidentally press the "Empty queue" button or delete a huge chunk of songs in the queue. In those cases it would be great if you could either press a key combination or a button to undo that.

**Comment by declension** [comment]
Yes... in fact _global_ undo was an idea brought up [as far back as 2.5](383) (see also #339).

I'm (more) keen to get it solved for #175 actually, but perhaps it's finally worth a reinvestigation.

**Comment by CreamyCookie** [comment]
@declension A global undo would of course be even better. Maybe we could create a new issue about global undo, add the relevant info and then close this and #175.
